README.md

```diff
@@ -1 +1,163 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello-Microverse <a name="about-project"></a>
+
+**Hello-Microverse** is a vanilla-css project for learning web development.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">
+      HTML <br>
+      CSS
+      </a>
+    </li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Added the styles for header**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Still not have live demo
+
```

Comment:
- [ ] Brilliant documentation! :clap: :clap: It is just that you missed a suggestion by the previous reviewer regarding this `Live demo` section that I think it is important to abide by. How about we remove this section completely, since this is not a page we ever plan to deploy? Even if we ended up deploying it, we can always update it, right? :thinking: Exactly! So, to fully make this repo professional, as will do with other future projects, let's remove this please :pleading_face: :pray:

Reply:
so should i remove only this text "Still not have live demo"?

Reply:
@MohammadYaser Sorry for the late response, since this project will never be deployed. I suggest you remove the `Live Demo` section completely from the table of content and the main body.